src/shader_recompiler/backend/spirv/emit_spirv_context_get_set.cpp

```diff
@@ -76,7 +76,11 @@ Id OutputAttrPointer(EmitContext& ctx, IR::Attribute attr, u32 element) {
     case IR::Attribute::Position0: {
         return ctx.OpAccessChain(ctx.output_f32, ctx.output_position, ctx.ConstU32(element));
     }
-    case IR::Attribute::Position1:
+    case IR::Attribute::Position1: {
+        ASSERT(ctx.stage == Stage::Geometry || ctx.stage == Stage::Vertex);
+        ASSERT_MSG(element == 2u, "Unsupported pos1 export");
```

Comment:
Hardcoding position1 as gl_Layer seems not too good idea. Does using VsOutputAttrPointer not work here like other exports? When it uses gl_Layer it should have value of VsOutput::GsMrtIndex